### DISCUSSION TITLE
Date Range  Slider in the DataFilterExtension

### DISCUSSION THREAD

**Discussion by kentstephen** [developmentseed/lonboard#510] [General]
Hi, 

I am trying to create a slider filter by date range in Lonboard. I think I am missing something conceptually, because it seems like the ```DataFilterExtension()``` wants floats or a numpy array. Is there any way to show geometries on different days with a slider? My dates look like ```20240101```. Using DuckDB. Thank you.

**Comment by kylebarron** [comment]
You need to convert your dates to floats. So you need to parse the string into a "date" type and then convert that date to timestamps.

However an extra point of complexity is that a float32 type can't represent milliseconds since unix epoch. So you usually want to subtract off the start of your date range, so you have "milliseconds since the start of your range". 

It would be great to make this simpler in the future

**Comment by kentstephen** [comment]
thank you Kyle, I probably will have to follow up on this. Appreciate your time.
